node/src/components/consensus/protocols/highway/performance_meter.rs

```diff
@@ -5,20 +5,21 @@ use tracing::error;
 
 use casper_types::{TimeDiff, Timestamp};
 
-use crate::components::consensus::{
-    highway_core::{
-        finality_detector::{
-            assigned_weight_and_latest_unit, find_max_quora, round_participation,
-            RoundParticipation,
+use crate::{
+    components::consensus::{
+        highway_core::{
+            finality_detector::{
+                assigned_weight_and_latest_unit, find_max_quora, round_participation,
+                RoundParticipation,
+            },
+            state, State,
         },
-        state, State,
+        traits::Context,
+        utils::ValidatorIndex,
     },
-    traits::Context,
-    utils::ValidatorIndex,
+    types::chainspec::PerformanceMeterConfig,
 };
 
-/// The number of most recent blocks for which we average the max quorum in which we participated.
-const BLOCKS_TO_CONSIDER: usize = 5;
 /// The average max quorum that triggers us to slow down: with this big or smaller average max
 /// quorum per `BLOCKS_TO_CONSIDER`, we increase our round length.
```

Comment:
Please update the comments to not mention `BLOCKS_TO_CONSIDER` (I think there are more that just this one).